src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -311,7 +311,7 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
 
   /** Helper method to filter REMOVED orgPersons from PeopleAndRoles list. */
   private filterRemovedOrgPersons (peopleAndRoles: Array<OrgPersonIF>): Array<OrgPersonIF> {
-    return peopleAndRoles.filter(orgPerson => !orgPerson.action?.includes(ActionTypes.REMOVED))
+    return peopleAndRoles.filter(orgPerson => !orgPerson.actions?.includes(ActionTypes.REMOVED))
```

Comment:
Does `includes` still work here? I see you changed it to `find` in Filings UI.

Reply:
I'm not sure about Filings UI. I think the recent change in Filings UI was related to roleTypes no? 
Was updated to use `some` i believe.

In this context, includes works as we are looking at an array of strings(ActionTypes) built for identifying the changes, or type of changes.

Reply:
You're right: Filings UI was roleType and this is action.

I think in both cases they're an array of enums. I believe `find(some)` and `includes` works the same for those.

No change needed here.

Reply:
Totally, i think if were dealing with a plain array (ie strings) i would default to includes, but some/find where we have to check the nested properties of objects in an array etc (ie { roleType:  someRole })